src/plugins/initiatives/initiativeFile.js

```diff
@@ -19,6 +20,28 @@ export const INITIATIVE_FILE_SUBTYPE = "INITIATIVE_FILE";
  */
 export type InitiativeFile = InitiativeFileV020;
 
+/**
+ * Stored inside a initiatives.json db file with other initiatives.
+ * presently the `id` prop is used as the key inside the db file
+ */
+export type InitiativeFileV100 = {|
+  id: string, // json file name right now. probably not needed if this is stored in side a json db file
+  // Compat info is usually stored within a header object, but in the db file parsing is more
+  // straightforward if it's folded inside the primary data object
+  type: string,
+  version: string,
```

Comment:
Can you explain why it's convenient to put it inside the primary data object? IMO it's a nice separation of concerns: once you know you have an InitiativeFileV100 you don't need a version or type info, because it's there in the type. So we include the type/version in the wrapper, but once you unwrap into a known type, you don't need that information in runtime any more.

Reply:
I was using this in the frontend to determine how to parse files, originally. Will remove this.